### ISSUE TITLE
"/click" doesn't pass expected "down" parameter to Button:Click()

### ISSUE THREAD

**Issue by Meorawr** [Stanzilla/WoWUIBugs#283]
The `/click` slash command doesn't correctly simulate clicks on secure action buttons due to an improper pattern match in [SecureCmdList["CLICK"]][CLICK] resulting in the `down` parameter passed to [Button:Click][Button:Click]() to be only ever be `false` if no mouse button name is specified, and always `true` otherwise.

---

1. Run `/console ActionButtonUseKeyDown 1` (this is the default).
2. Run the following snippet in the chat frame:
```
/run CreateFrame("Button", "B", nil, "SecureActionButtonTemplate, UIPanelButtonTemplate") B:SetAttribute("type", "macro") B:SetAttribute("macrotext", "/run print('Boo!')") B:SetPoint("CENTER")
```
3. Run `/click B`. The message "Boo!" will be printed to the chat frame.
4. Run `/click B LeftButton`. Nothing will be printed to the chat frame.

[Button:Click]: https://wowpedia.fandom.com/wiki/API_Button_Click
[CLICK]: https://github.com/Gethe/wow-ui-source/blob/c851b385acc54a08438ecbc639a9ae61be353c51/Interface/FrameXML/ChatFrame.lua#L1606-L1612

**Comment by Meorawr** [comment]
Still somewhat an issue in 10.0.0.46047, but a bit different - the test now does the opposite of what was originally stated. Shortened the ticket to make it more relevant.

**Comment by Nevcairiel** [comment]
The issue has returned to the original behavior, the inverse as described above was caused by another bug that has since been fixed.

**Comment by Meorawr** [comment]
Closing as semi-fixed in 10.0.2.46479; the argument processing aspect should be improved in that `/click global button` won't incorrectly set the state of the `down` parameter, and `/click global button 0|1` would expectedly control the `down` parameter.

What isn't resolved is the default click type with respect to click type registrations (#317), will make a ticket to get this looked at when progress is made on #317.
